tajo-common/src/main/java/org/apache/tajo/conf/TajoConf.java

```diff
@@ -315,6 +315,9 @@ public static int setDateOrder(int dateOrder) {
 
     $GROUPBY_MULTI_LEVEL_ENABLED("tajo.dist-query.groupby.multi-level-aggr", true),
 
+    // WARN "tajo.yarn-rm.parallel-task-runner-launcher-num" should be set enough to avoid deadlock
+    $QUERY_EXECUTE_PARALLEL_MAX("tajo.query.execute.parallel.max", 2),
```

Comment:
There are a lot of remaining issues to fully support concurrent stage execution. One of the representative issues is resource scheduling. However, as you already know, our new resource scheduler which can schedule resources for multiple queries and stages is on-going work. So, we cannot handle this issue now.
Thus, even though this feature is still nice, it should be disabled by default. In other words, the default value of parallelism should be 1.

Additionally, this is related to the concurrent execution of multiple stages. So, it would be better if the name is intuitive. How about ```tajo.querymaster.stage.parallel-execution.max-num```?